Service Bus message size

andynava opened the pull request:
Service Bus supports messages up to 1 MB in Premium tier: https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quotas

azurecla commented:
Hi __@sahaj08__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sahajs). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
@barbkess 
Please review and approve here, thanks.

barbkess commented:
Please close.  I submitted a different request with the fix that adds an empty line before and after the ``` .

andynava committed:
Service Bus message size

Service Bus supports messages up to 1 MB in Premium tier: https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quotas

GitHubber17 committed:
Update functions-best-practices.md
